docker-compose.yml

```diff
@@ -37,5 +37,19 @@ services:
     ports:
       - ${ADMINER_PORT}:8080
 
+  swagger-ui:
+    image: swaggerapi/swagger-ui
+    container_name: swagger-ui
+    ports:
+        - "80:8080"
+    volumes:
+        - ./docs:/usr/share/nginx/html/docs
+    environment:
+        API_URL: docs/openapi.yaml
+        # For multiple specifications
+        # URLS: "[{ url: \"docs/first_openapi.yaml\", name: \"FirstApiSpec\" },
+        #         { url: \"docs/second_openapi.yaml\", name: \"SecondApiSpec\" },
+        #     ]"
+
```

Comment:
@Cyantosh0 
Let's read port mapping from the environment variable as done in other services. Let's remove the comment in the environment as well.

Reply:
Other things look fine to me. Thank You. 🙇

Reply:
@dipeshdulal sir, I have updated things as per your request. Please let me know, if there's anything.  🙇